### PR TITLE
Update `EXAMPLE` sections for multiuser and multihost handling.

### PR DESCRIPTION
**Pull request by dkarpele**
Solves #1040

### CODE REVIEW

File: plugins/modules/ipauser.py [hunk 1]
```diff
@@ -573,6 +584,14 @@
     name: pinky,brain
     state: enabled
 
+# Remove but preserve user pinky
```
Comment (rjeffman) [inline plugins/modules/ipauser.py:587]: We can remove several users without resorting to `users` simply by adding multiple names. In my opinion, that's a more readable way to do this.

@t-woerner what do you think?

Comment (t-woerner) [reply plugins/modules/ipauser.py:587]: Yes, I agree. It is working both ways, though. Therefore it might be good to have both examples.

**Review by rjeffman — APPROVED** [review]
lgtm

**Review by t-woerner — APPROVED** [review]
LGTM

**Pull request merged**